src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -4,7 +4,6 @@ import PropTypes from 'prop-types';
 import {StyleSheet, Text, View} from 'react-native';
 import {useNavigation, useRoute} from '@react-navigation/native';
 
-import {useNetInfo} from '@react-native-community/netinfo';
 import DiscoveryAction from '../../../context/actions/discoveryAction';
 import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
 import DomainList from '../elements/DiscoveryItemList';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useNetInfo` import from `'@react-native-community/netinfo'` has been removed. If this was used to handle network connectivity issues, its removal might cause problems when the device is offline. Please ensure that network connectivity is handled properly elsewhere in your code.
```diff
- import {useNetInfo} from '@react-native-community/netinfo';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->